client/src/components/scoresheet/scoresheet.js

```diff
@@ -1,51 +1,55 @@
 'use strict'
 
 export default {
-	template: `
-	<div class="top-bar secondary">
-		<div class="top-bar-left">
-			<ul class="menu">
-				<li><ref-identity ng-if="scoresheet.isRef"></ref-identity></li>
-				<a editable-select="scoresheet.scoresheet.teamNumber" buttons="no" blur="submit"
-					class="menu-text" onaftersave="scoresheet.processErrors()"
-					e-ng-options="team.number as team.displayText for team in scoresheet.teams">
-					{{ scoresheet.teamText() }}
-				</a>
-			</ul>
-		</div>
-		<div class="top-bar-right flex-container">
-			<ul class="menu">
-				<il>
-					<span id="score-diff-animation" ng-show="isFinite(scoresheet.showingScoreDiffAnimation)">{{scoresheet.scoreDiff}}</span>
-				</il>
-				<il><div class="hollow button">{{scoresheet.score()}} pts.</div></il>
-				<il><div class="button" ng-if="scoresheet.isAdmin" ng-click="scoresheet.setDefault()">
-					<i class="fa fa-arrow-down"></i>
-				</div></il>
-				<il><div class="button" ng-click="scoresheet.reset()"><i class="fa fa-undo"></i></div></il>
-			</ul>
-		</div>
+	template: `<div class="top-bar secondary">
+	<div class="top-bar-left">
+		<ul class="menu">
+			<li><ref-identity ng-if="scoresheet.isRef"></ref-identity></li>
+			<form>
+				<input type="text" list="teams" ng-model="scoresheet.scoresheet.teamNumber" onaftersave="scoresheet.processErrors()">

```

Comment:
The attribute `onaftersave` has no meaning on an `<input>` tag. Please remove it and find another way for the validation to run every time. You can use `ng-change` perhaps.